src/components/SendTokenDialog.vue

```diff
@@ -615,6 +649,82 @@ export default defineComponent({
       this.showDeleteDialog = false;
       this.clearAllWorkers();
     },
+    writeTokensToCard: function () {
+      if (!this.scanningCard) {
+        try {
+          this.ndef = new window.NDEFReader();
+          this.controller = new AbortController();
+          const signal = this.controller.signal;
+          this.ndef
+            .scan({ signal })
+            .then(() => {
+              console.log("> Scan started");
+
+              this.ndef.onreadingerror = (error) => {
+                console.error(`Cannot read NDEF data! ${error}`);
+                notifyError("Cannot read data from the NFC tag");
+                this.controller.abort();
+                this.scanningCard = false;
+              };
+
+              this.ndef.onreading = ({ message, serialNumber }) => {
+                console.log(`Read card ${serialNumber}`);
+                notify(`Serial: ${serialNumber}`);
+                this.controller.abort();
+                this.scanningCard = false;
+                try {
+                  const arrBuffer = new TextEncoder().encode(
+                    this.sendData.tokensBase64
+                  );
+                  const ndefRecord = new NDEFRecord({
+                    data: arrBuffer,
```

Comment:
ISTM that NDEFRecord.data [can also be a string](https://developer.mozilla.org/en-US/docs/Web/API/NDEFReader/write#data), so maybe we can simplify to this?

```suggestion
                  const ndefRecord = new NDEFRecord({
                    data: this.sendData.tokensBase64,
```

Or have you tried that this does not work?

Reply:
I had tried this before and also now and it doesn't seem to work. Says it wants an ArrayBuffer or ArrayBufferView.

Reply:
Thanks for confirming